l2os/drivers/l2output/driver.go

```diff
@@ -241,3 +243,17 @@ func (d *Driver) SendTransaction(
 
 	return d.cfg.L1Client.SendTransaction(ctx, tx)
 }
+
+func (d *Driver) outputRootAtBlock(ctx context.Context, blockNum *big.Int) (l2.Bytes32, error) {
```

Comment:
small nit: perhaps it's worth exporting this method (and passing in `RollupClient`) so that we can use this directly in the e2e tests?

down the road, and depending on how many rpcs we end up with, it could make sense to wrap the `rpc.Client` with a dedicated struct that exposes a method for each endpoint